master/internal/experiment/bulk_action.go

```diff
@@ -182,12 +198,12 @@ func ToAPIResults(results []ExperimentActionResult) []*apiv1.ExperimentActionRes
 
 // ActivateExperiments works on one or many experiments.
 func ActivateExperiments(ctx context.Context,
-	experimentIds []int32, filters *apiv1.BulkExperimentFilters,
+	experimentIds []int32, filters *apiv1.BulkExperimentFilters, searchFilter *string,
 ) ([]ExperimentActionResult, error) {
 	if filters != nil && filters.States == nil {
 		filters.States = []experimentv1.State{experimentv1.State_STATE_PAUSED}
 	}
```

Comment:
It looks like work is being done here to correct the filter to only activate paused experiments, I'm not under the impression the same is being done for `SearchFilters`, this seems like a regression.

Reply:
https://github.com/determined-ai/determined/assets/25434156/06aa1a61-b931-4fa1-ad70-324acfa837d2

Reply:
There are only 106 experiments; however, we get an error that says it failed on 242 experiments.

Reply:
This is the downside of having searchFilters being setup as a string like this is, we don't don't how it's structured until later on when the parsing logic is specified

Reply:
I added a check to add in a state filter on relevant endpoints when searchFilter is being used & filter is not.

Reply:
IMO this is a bandaid solution for a larger problem with how we approach bulk endpoints